### PR TITLE
release v1.6.0 release notes

### PR DESCRIPTION
**Pull request by alexlyp**

### CODE REVIEW

**Review by jrick — APPROVED** [review]

File: release-notes.md [hunk 1]
```diff
@@ -1,3 +1,1231 @@
...
+
+
+# decrediton v1.6.0-rc1
+
+This release includes some great new features and also includes a large 
+portion of refactor of the codebase that will, hopefully, lead to more
+efficient development in the future.
+
+We are proud to include the first iteration of our privacy implementation in 
+this release.  Users may not set up their wallet to be a "privacy" wallet
```
Comment (xaur) [inline release-notes.md:1075]: may not -> may now

**Review by davecgh — APPROVED** [review]

File: release-notes.md [hunk 2]
```diff
@@ -1,3 +1,1790 @@
...
+- blockchain/standalone: Prepare v2.0.0 ([decred/dcrd#2395](https://github.com/decred/dcrd/pull/2395))
+- blockchain/stake: Prepare v3.0.0 ([decred/dcrd#2418](https://github.com/decred/dcrd/pull/2418))
+- gcs: Prepare v2.1.0 ([decred/dcrd#2420](https://github.com/decred/dcrd/pull/2420))
+- peer: Prepare v2.2.0 ([decred/dcrd#2422](https://github.com/decred/dcrd/pull/2422))
+- rpcclient: Prepare v6.0.0 ([decred/dcrd#2423](https://github.com/decred/dcrd/pull/2423))
+- blockchain: Prepare v3.0.0 ([decred/dcrd#2424](https://github.com/decred/dcrd/pull/2424))
+- rpcclient: Prepare v6.0.1 ([decred/dcrd#2455](https://github.com/decred/dcrd/pull/2455))
+- main: Update to use all new module versions ([decred/dcrd#2426](https://github.com/decred/dcrd/pull/2426))
+- main: Remove module replacements ([decred/dcrd#2428](https://github.com/decred/dcrd/pull/2428))
+- main: Use backported module up
```
Comment (xaur) [inline release-notes.md:816]: dcrd release notes got truncated. Two commit listings and Code Contributors sections are now missing.

Comment (xaur) [reply release-notes.md:816]: Fix to restore the truncated dcrd section pushed alogn other tweaks in https://github.com/alexlyp/decred-binaries/pull/1

**Comment by vanjan** [comment]
time to merge!

**Pull request merged**